index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>A Meaningful Page Title</title>
+    <link href="style.css" rel="stylesheet" />
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
+        <img id="image" src="images/10613180113_fdf7bcd316_b.jpg" />
```

Comment:
- [ ] It is better to use `classes` instead of `ids` .
Kindly change `ids` to be `classes`

Reply:
I think it would be good to explain why classes should be used instead of ids, so the same mistake can be avoided next time.